# try:
#   num = int(input("please enter a number: "))
#   print(num)
# #
# except Exception as e:
#     print("Invalid number. Please try again.")
try:
  num = int(input("please enter a number: "))
  print(num)
except:
  print("dInvalid entry: please try agin")
